src/audio/google/google_ctc_audio_processing.c

```diff
@@ -295,32 +316,21 @@ static int google_ctc_audio_processing_reconfigure(struct processing_module *mod
 	}
 
 	comp_info(dev, "google_ctc_audio_processing_reconfigure(): New tuning config %p (%zu bytes)",
-		  config, size);
+		  cd->config, size);
 
-	cd->reconfigure = false;
+	cd->config = (struct google_ctc_config *)config;
+	if (cd->config)
```

Comment:
this check is redundant - `config` has been checked to be non-`NULL` 9 lines above